### DISCUSSION TITLE
[Question][Hw1] Padding Oracle Server on Test Environment

### DISCUSSION THREAD

**Discussion by luaneyed** [prosyslab-classroom/cs348-information-security#491] [Q&A]
Can we suppose the padding oracle server is listening localhost:5000 on test environment?

**Comment by Hwi-gyeom** [comment]
the code will set proper port number. Therefore, no.

**Comment by luaneyed** [comment]
@Hwi-gyeom 

Then can we suppose there is `oracle_python_v1_2.py` in same directory?

**Comment by Hwi-gyeom** [comment]
yes
